src/include/wiredtiger.h.in

```diff
@@ -5712,6 +5732,58 @@ struct __wt_page_log {
     int (*terminate)(WT_PAGE_LOG *page_log, WT_SESSION *session);
 };
 
+/*
+ * A structure that represents encrypted keys stored by WiredTiger.
+ *
+ * The data field is a flexible array member for storing variable-length
+ * data in a contiguous memory block.
+ */
+struct __wt_crypt_key {
+    uint64_t result;
+    size_t size;
```

Comment:
Does the size include result/size?  (I think the answer is yes from the example usage).  Either way, it should be stated in a comment.

Reply:
I put this in a different comment on the updated code, but if WT is not able to load the blob, the load function will never be called.  So there's no need for a result here.